### DISCUSSION TITLE
Multiple features

### DISCUSSION THREAD

**Discussion by 9mido** [fonoster/yaps#251] [Ideas]
Holding:
Put the user on hold with waiting music/recording before a call starts and during the call
When on hold, be able to make another call with someone else but still be able to return back to the on hold caller or put them in a queue for someone else (parking).

Transfers:
Warm/cold transfers to another phone number
There should be a fallback in case of getting stuck where the number we are transferring to does not answer.

Conference:
Adding multiple people to a single call
Password for entering into the conference

Extensions:
Dial a phone number extension

Call statuses:
There should be a way for a user to see who is calling them, if the call is disconnected, busy, on hold, mute, etc.

**Comment by thebasicproducts** [comment]
Would love 

CRM built-in kind of like Bitrix24
Sms messaging 
end to end encryption
integrations like Onlyoffice, Mega.nz
